snapshot-labs/sx-monorepo
# fix(ux): make all sticky nav horizontally scrollable

### Summary



On narrow screen, the proposal top nav is preventing the content from resizing correctly

Before fix

![Screenshot 2024-08-31 at 01 32 07](https://github.com/user-attachments/assets/e32c8feb-2c6c-471f-bda8-9a3b4237a45f)

After fix

![Screenshot 2024-09-04 at 13 09 49](https://github.com/user-attachments/assets/88a87790-1112-4d00-8921-fd3bb7368662)


This PR will:

- make the proposal page top sticky navigation horizontally scrollable, like all other top sticky nav
- fix all sticky nav to have the same style (same gradient and with buttons, like the space settings nav)

### How to test

1. Visit any pages with sticky nav (proposal, space user, treasuries, etc ...)
2. Resize the screen, and decrease width
3. The sticky nav should be horizontally scrollable on narrow screen, but buttons and gradient
